packages/p2p/src/components/verification/__tests__/verification.spec.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { useStores } from 'Stores';
+import Verification from '../verification.jsx';
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(),
+}));
+
+const mockProps = () => ({
+    should_wrap: false,
```

Comment:
@artsemkavaleu-deriv maybe it will be better to use:
const props = { your props } instead of invoke mockProps function in every test case,